src/python/python.cpp

```diff
@@ -433,7 +433,8 @@ PYBIND11_MODULE(onnxruntime_genai, m) {
       .def(pybind11::init([](const std::string& config_path) { return OgaConfig::Create(config_path.c_str()); }))
       .def("append_provider", &OgaConfig::AppendProvider)
       .def("set_provider_option", &OgaConfig::SetProviderOption)
-      .def("clear_providers", &OgaConfig::ClearProviders);
+      .def("clear_providers", &OgaConfig::ClearProviders)
+      .def("get_model_type", &OgaConfig::GetModelType);
```

Comment:
Maybe we can define this as a `def_property_readonly`?

Reply:
Thanks, replaced.